src/Symfony/Component/Validator/Tests/Constraints/IbanValidatorTest.php

```diff
@@ -133,6 +133,7 @@ public function getValidIbans()
             array('FR7630007000110009970004942'), //Central African Republic
             array('CG5230011000202151234567890'), //Congo
             array('CR0515202001026284066'), //Costa Rica
```

Comment:
So this example is wrong actually, no?

Reply:
It should be wrong, yes. But I keep it because BC.

Reply:
There is no BC required here I think, it's clearly a bug fix 🤔

Reply:
So I can just make it required with 0 and commit again? Or just leave it like that?

Reply:
https://www.ibancalculator.com says the checksum is wrong. So yes, I think we should remove this example.

Reply:
@xabbuh @sroze old format is now removed. Validator accepts only the right one now.